resolwe_bio/processes/import_data/genome.yml

```diff
@@ -170,7 +172,8 @@
         format_index_files.py ${NAME}.tar.gz ${FASTA_NAME}_index bowtie2_index
       {% else %}
         mkdir bowtie2_index
-        bowtie2-build "$FASTA_NAME.fasta" "bowtie2_index/${FASTA_NAME}_index"
+        bowtie2-build "$FASTA_NAME.fasta" "bowtie2_index/${FASTA_NAME}_index" \
+        --threads {{ requirements.resources.cores }}
```

Comment:
Put each input parameter in it's own line. Also, indent parameters. Do it also for the upper call to `bowtie-build`.